docs/feature-store/using-spark-engine.md

```diff
@@ -238,4 +268,73 @@ target = ParquetTarget(
 )
 
 fstore.ingest(feature_set, source, targets=[target], run_config=run_config, spark_context=spark_service_name)
-```
\ No newline at end of file
+```
+
+## Spark ingestion from Snowflake example
+
+Spark ingestion from Snowflake can be executed both remotely and locally. The following code executes local data ingestion from Snowflake.
+
+```
+from pyspark.sql import SparkSession
+
+import mlrun
+import mlrun.feature_store as fstore
+from mlrun.datastore.sources import SnowflakeSource
+
+spark = SparkSession.builder.appName("snowy").getOrCreate()
+
+mlrun.set_environment(project="feature-store")
+feature_set = fstore.FeatureSet(
+    name="customer", entities=[fstore.Entity("C_CUSTKEY")], engine="spark"
+)
+
+source = SnowflakeSource(
+    "customer_sf",
+    query="select * from customer limit 100000",
+    url="<url>",
+    user="nnnnnn",
+    password="nnnnnn",
```

Comment:
```suggestion
    user="<username>",
    password="<password>",
```

If we're using it for the `<url>` parameter, might just as well stick to the convention.